dev-docs/development-process.md

```diff
@@ -160,6 +160,14 @@ Now follow along:
    followed by `dev:install`, instead.
 1. `yarn dev:uninstall` will uninstall your deployment.
 
+Troubleshooting:
+* Certbot error `[Errno 13] Permission denied: '/var/log/letsencrypt' Either run as root, or set --config-dir, --work-dir, and --logs-dir to writeable paths.` - do not run as root, but set the directories instead
+* Dev config creation step: `AccessDenied: Access Denied` error with a stack trace pointing at aws-sdk library - make sure to run [this script](https://gist.github.com/djmitche/80353576a0f389bf130bcb439f63d070). Note that you'll be running that script many times, so it's wise to setup the function (see the starting comment in the script) or an alias. This script fetches the temporary credentials for aws and saves them to env variables.
```

Comment:
Since this is general instructions even for people who are outside of our team, should we still say this? Maybe something more general